### ISSUE TITLE
False-alarm in `RELAXED_DECORATORS` detection breaks compatibility with Python < 3.9 

### ISSUE THREAD

**Issue by Zac-HD** [psf/black#2181]
Running Black 21.4b2 on the following code:

```python
@get_decorator()
def f():
    return u"This should be Unicode on Python 2"
```

incorrectly strips the `u`-prefix from the string, because the minimum Python version is incorrectly detected as 3.9 (via a false detection of relaxed decorator syntax).  Removing the `()` in `@get_decorator()` avoids this problem.

I'm pretty sure that the solution is to fix [`black.is_simple_decorator_expression()`](https://github.com/psf/black/blob/35e8d1560d68e8113ff926a9f832582f8f4a694f/src/black/__init__.py#L5775-L5796).

**Comment by graingert** [comment]
Can you pin this issue until a fix is released?

**Comment by ambv** [comment]
Released in 21.5b0 so unpinning.
